src/query/src/dist_plan/planner.rs

```diff
@@ -229,24 +215,3 @@ impl TreeNodeVisitor for TableNameExtractor {
         }
     }
 }
-
-struct TableNameRewriter;
```

Comment:
Why we don't need to rewrite the table name?

Reply:
It seems it's not needed at the first place. Substrait is able to serde the fully qualified table name correctly. Rewriting table name like this could make physical plan failed at the optimization stage because the schema is not match before and after.